commands/uploadphotos.js

```diff
@@ -0,0 +1,57 @@
+import { google } from "googleapis"
+import { authorize } from "../utilities/google-auth.js"
+import fs from "fs"
+import { ContextMenuCommandBuilder, ApplicationCommandType } from "discord.js"
+import axios from "axios"
+import path from "path"
+
+export const data = new ContextMenuCommandBuilder().setName("upload photos").setType(ApplicationCommandType.Message)
+
+export async function execute(interaction) {
+  if (!interaction.isMessageContextMenuCommand()) return
+  const message = await interaction.channel.messages.fetch(interaction.targetId)
+  await interaction.deferReply() // Discord requires an acknowledgement within 3 seconds. this allows the response to be deferred, with an "<application> is thinking..." response in the meantime
+
+  let auth = await authorize()
+  const drive = google.drive({ version: "v3", auth })
+  const folderId = "[FOLDER ID]" // replace this with the id of the desired folder (the part of the url after "folders/")
```

Comment:
Ideally we should not hardcode config values - use an environment variable instead, which can be set in the `.env` file